src/transformers/utils/hub.py

```diff
@@ -766,7 +945,7 @@ def has_file(
 
     r = requests.head(url, headers=headers, allow_redirects=False, proxies=proxies, timeout=10)
     try:
-        _raise_for_status(r)
+        huggingface_hub.utils._errors._raise_for_status(r)
```

Comment:
Nice :)

Reply:
Only used it in this function because it is there to stay, the other ones still use the old `_raise_for_status` but they will disappear soon.